src/Solver.php

```diff
@@ -55,8 +55,14 @@ public function solve(string $stringA, string $stringB)
             return call_user_func_array([$this, 'solve'], $arguments);
         }
 
-        $charsA = str_split($stringA);
-        $charsB = str_split($stringB);
+        $charsA = [];
+        $charsB = [];
+        for ($i=0; $i < max(mb_strlen($stringA), 1); $i++) {
```

Comment:
I appreciate that this loop is present due to a lack of `mb_str_split()`.

Can this be logic be factored out to a method (perhaps named `mb_str_split()` so that the code at this point reads more clearly?